Package.swift

```diff
@@ -5,27 +5,39 @@ import PackageDescription
 
 let package = Package(
     name: "Web3SwiftMpcProvider",
-    platforms: [.iOS(.v13), .macOS(.v11)],
+    platforms: [.iOS(.v14), .macOS(.v11)],
     products: [
         .library(
-            name: "Web3SwiftMpcProvider",
-            targets: ["Web3SwiftMpcProvider"]),
+            name: "MPCEthereumProvider",
+            targets: ["MPCEthereumProvider"]),
+        .library(
+            name: "MPCBitcoinProvider",
+            targets: ["MPCBitcoinProvider"]),
+
 
     ],
     dependencies: [
         .package(url: "https://github.com/argentlabs/web3.swift", from:"1.6.0"),
-        .package(url: "https://github.com/torusresearch/tss-client-swift.git", from: "2.0.0"),
+        .package(url: "https://github.com/tkey/mpc-core-kit-swift", .exact("1.0.0-alpha")),
```

Comment:
This inverts the dependencies.

This library is supposed to be consumed by mpc-core-kit, not the other way round.

Since you'll not get an optional import of multiple providers, they will be bundled together, I'd suggest looking into the facade design pattern in swift for how to properly wrap this in mpc-core-kit. This will also eliminate problems as seen in #10 

You also need to tag the releases with a semver version, the "-alpha" tag will cause issues.